src/Sarif.Multitool.Library/Rules/ADO1001MaximumRunsCount.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool.Rules
+{
+    public class ADO1001MaximumRunsCount : Base1001MaximumRunsCount
```

Comment:
>ADO1001 [](http://example.com/codeflow?start=17&length=7)

Should we add rule id into class name?  The existing validation rule names don't contain the rule id

Reply:
The problem is going to be the fact that we have the same rule name for two services, ADO and GHAS. I can drop the rule ID but will have to keep the service name.